composer.json

```diff
@@ -350,19 +350,16 @@
             "drupal/entity_hierarchy": {
                 "Hidden Weight field causes extra space in the form": "patches/entity_hierarchy-weight-widget-space-3245879-2.patch",
                 "Fatal error on node/{nid}/media #3034602": "https://www.drupal.org/files/issues/2021-11-25/issue-3034602_fatal_error_on_getCandidateFields.patch",
-                "Async support & save only changed, based on https://www.drupal.org/files/issues/2021-05-06/3178313-tree-editor-3.patch) but only save parent changes when saving on the children tab": "patches/DP-23411_node-hierarchy-async-support-save-only-changed--v4.patch"
+                "Async support & save only changed, based on https://www.drupal.org/files/issues/2021-05-06/3178313-tree-editor-3.patch) but only save parent changes when saving on the children tab": "patches/DP-23411_node-hierarchy-async-support-save-only-changed--v4.patch",
+                "Enable entity_hierarchy_breadcrumb module to work on node edit routes": "patches/entity_hierarchy_breadcrumb_for_admin_routes.patch"
             },
             "drupal/entity_usage": {
                 "2954973 - Track absolute URLs in link fields": "https://www.drupal.org/files/issues/2021-08-17/2954973-63.patch",
                 "3007752 - Entity usage list does not scale (https://www.drupal.org/project/entity_usage/issues/3007752)": "patches/entity_usage-usage_list_scale-3007752-17.patch",
                 "Hides language column and shows bundle" : "patches/entity-usage--hide-language--show-bundle.patch",
                 "3254437 - Delete queue items for `entity_usage_regenerate_queue` when `running drush entity-usage:recreate`  ": "https://www.drupal.org/files/issues/2021-12-14/issue-3254437--remove-items-from-queue-before-recreating-it.patch",
-                "3015287 - Allow usage records to be registered in background" : "https://www.drupal.org/files/issues/2019-07-22/3015287-12.patch",
```

Comment:
@beto-aveiga This change looks OK to you?

Reply:
@deviantintegral See comment above ... You can ignore my comment about the missing API call to switch PHP version. I forgot that went in separately.